zebedee/client/client_test.go

```diff
@@ -19,21 +21,22 @@ func TestUnitClient(t *testing.T) {
 	go mockZebedeeServer(portChan)
 
 	port := <-portChan
+	//rchttpCli := rchttp.NewClient()
+	cli := NewZebedeeClient(rchttp.NewClient(), fmt.Sprintf("http://localhost:%d", port))
 
-	cli := NewZebedeeClient(fmt.Sprintf("http://localhost:%d", port))
-
-	ctx := context.WithValue(context.Background(), "X-Florence-Token", "test-access-token")
+	testAccessToken := "test-access-token"
+	ctx := context.Background()
 
 	Convey("test get()", t, func() {
 		Convey("test get sucessfully returns response from zebedee", func() {
-			b, err := cli.get(ctx, "/data?uri=foo")
+			b, err := cli.get(ctx, testAccessToken, "/data?uri=foo")
```

Comment:
You are going to hate me for this _but_ I think each of these updated tests should also assert that the outbound request has the header set.